app/static/openapi.yaml

```diff
@@ -722,6 +741,96 @@ components:
             message:
               description: Error message
               type: string
+    Error_list:
+      type: object
+      properties:
+        apiVersion:
+          type: string
+          description: API version
+        error:
+          type: integer
+          description: 0 on successful operation, 1 on failure
```

Comment:
why have a flag field for this? shouldn't the rest response code give you this? or the presence of another field that contains the errors? also, if you are going to have a flag field, why would you use a number instead of a boolean?

Reply:
I have no idea, that one was something I added based on your suggestion in slack. If it makes no sense to you, then I guess I misunderstood what you were suggesting in that message. I'm fine taking this boolean flag out altogether because it makes no sense to me why its in there.

Reply:
Per slack conversation, I'm going to remove this flag altogether from the API